File: Task-3/Java/SubTask-2.java
import java.io.*;

public class FileReadWrite {
    public static void main(String[] args) {
        try {
            BufferedReader a = new BufferedReader(new FileReader("input.txt"));
            BufferedWriter b = new BufferedWriter(new FileWriter("output.txt"));
            
            String line = a.readLine();
            if (line != null) {
                b.write(line);
            }
            
            a.close();
            b.close();
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: Task-3/Ruby/SubTask-2.rb
input_file = 'input.txt'
output_file = 'output.txt'

begin
  input = File.open(input_file, 'r')
  output = File.open(output_file, 'w')

  input.each_line do |line|
    output.write(line)
  end
ensure
  input.close if input
  output.close if output
end
